releases/release-1.26/README.md

```diff
@@ -39,13 +39,14 @@ The 1.26 release cycle is as follows:
 
 - **Monday 5th September 2022**: Week 1 — Release cycle begins
 - **Thursday 29th September 2022**: Week 4 — [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
-- **01:00 UTC Friday 7th October 2022** / **18:00 PDT Thursday 6th October 2022** : Week 5 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
-- **Wednesday 27th July 2022**: Week 11 — [Release Retrospective][Retrospective Document] part 1
-- **01:00 UTC Wednesday 23rd November 2022** / **02:00 BST Wednesday 23rd November 2022** / **18:00 PDT Tuesday 22nd November 2022**: Week 12 — [Code Freeze](../release_phases.md#code-freeze)
-- **01:00 UTC Wednesday 30th November 2022** / **02:00 BST Wednesday 30th November 2022** / **18:00 PDT Tuesday 29th November 2022**: Week 13 — [Test Freeze](../release_phases.md#test-freeze)
-- **Tuesday 6th December 2022**: Week 14 — Docs must be completed and reviewed
-- **Tuesday 13rd December 2022**: Week 15 — Kubernetes v1.26.0 released
-- **TBC**: Week 16 — [Release Retrospective][Retrospective Document] part 2
+- **Friday 7th October 2022**: Week 5 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
```

Comment:
Would be great to have detailed time listed in summary to avoid confusion on time zone.

Reply:
Yes! good catch

Reply:
https://github.com/kubernetes/sig-release/pull/2012